src/main/webapp/ScanSettings.js

```diff
@@ -143,8 +144,10 @@ class ScanSettings {
 
         rows.hide();
         this.hideMessages();
+        this.isBsi = false;
```

Comment:
Is this change for the same defect as in the title ?

Reply:
Yes, the way it initializes in older versions it seemed to not matter. This way works for older and newest Jenkins UI